tests/components/withings/__init__.py

```diff
@@ -1 +1,85 @@
 """Tests for the withings component."""
+from collections.abc import Iterable
+from typing import Any, Optional
+from urllib.parse import urlparse
+
+import arrow
+from withings_api import DateType
+from withings_api.common import (
+    GetSleepSummaryField,
+    MeasureGetMeasGroupCategory,
+    MeasureGetMeasResponse,
+    MeasureType,
+    SleepGetSummaryResponse,
+    UserGetDeviceResponse,
+)
+
+from homeassistant.components.webhook import async_generate_url
+from homeassistant.core import HomeAssistant
+
+from .common import ProfileConfig, WebhookResponse
+
+
+async def call_webhook(
+    hass: HomeAssistant, webhook_id: str, data: dict[str, Any], client
+) -> WebhookResponse:
+    """Call the webhook."""
+    webhook_url = async_generate_url(hass, webhook_id)
+
+    resp = await client.post(
+        urlparse(webhook_url).path,
+        data=data,
+    )
+
+    # Wait for remaining tasks to complete.
+    await hass.async_block_till_done()
+
+    data = await resp.json()
```

Comment:
Is there a data type we can use for the variable `data`? Right now, it will use duck typing but assigning a predictable type makes it much easier to code with in the future.
```
data: MyResponseType = await resp.json()
```

Reply:
Since WebhookResponse is a class from the Withings tests, I'll have a look later at other integrations how they handle webhooks in tests and apply that knowledge here. For now it will be a `dict[str, Any]`